static/src/javascripts/projects/common/utils/$.js

```diff
@@ -24,11 +26,25 @@ define([
             return $.ancestor(el.parentNode, c);
         }
     };
+
+    $.isDescendantOrSelf = function (el, ancestor) {
+        if (!el || !ancestor || !el.parentNode || el.nodeName.toLowerCase() === 'html') {
+            return false;
+        }
+
+        if (el === ancestor || el.parentNode === ancestor) {
```

Comment:
Why do you check `parentNode` here? Won't it be checked in the next call of the function?